napari/layers/image/_tests/test_image.py

```diff
@@ -565,26 +565,42 @@ def test_value():
 
 
 @pytest.mark.parametrize(
-    ('position', 'view_direction', 'dims_displayed', 'world'),
+    (
+        'position',
+        'view_direction',
+        'dims_displayed',
+        'world',
+        'render_mode',
+        'result',
+    ),
     [
-        ((0, 0, 0), [1, 0, 0], [0, 1, 2], False),
-        ((0, 0, 0), [1, 0, 0], [0, 1, 2], True),
-        ((0, 0, 0, 0), [0, 1, 0, 0], [1, 2, 3], True),
+        ((0, 0, 0), [1, 0, 0], [0, 1, 2], False, 'mip', 0),
+        ((0, 0, 0), [1, 0, 0], [0, 1, 2], True, 'mip', 0),
+        ((2, 2, 2), [1, 0, 0], [0, 1, 2], False, 'mip', 1),
+        ((2, 2, 2), [1, 0, 0], [0, 1, 2], False, 'minip', 0),
+        ((2, 2, 2), [1, 0, 0], [0, 1, 2], False, 'average', 1 / 5),
+        ((2, 2, 2), [1, 0, 0], [0, 1, 2], False, 'translucent', 0),
+        # not quite as expected for additive
+        ((2, 2, 2), [1, 0, 0], [0, 1, 2], False, 'additive', 2),
+        ((2, 2, 2), [1, 0, 0], [0, 1, 2], False, 'iso', 0.5),
+        ((2, 2, 2), [1, 0, 0], [0, 1, 2], False, 'attenuated_mip', 0.81873075),
+        ((0, 2, 2, 2), [0, 1, 0, 0], [1, 2, 3], False, 'mip', 1),
```

Comment:
Should we test some other view directions? Oblique angles may be hard to confirm correct values I suppose.